pkgs/applications/virtualization/virt-top/default.nix

```diff
@@ -15,10 +15,14 @@ stdenv.mkDerivation rec {
 
   src = fetchgit {
     url = "git://git.annexia.org/virt-top.git";
-    rev = "v${version}";
+    tag = "v${version}";
```

Comment:
I get "No route to host" to this git host, and cannot verify that `tag` works. Please revert this line